serve/mlc_serve/model/paged_cache_model.py

```diff
@@ -35,6 +34,49 @@
 LOG = structlog.stdlib.get_logger(__name__)
 
 
+def fetch_raw_logprob_infos(
+        logits,
+        res_tokens,
+        sampling_params,
+) -> List[Optional[RawLogprobsInfo]]:
+    logprob_infos: List[Optional[RawLogprobsInfo]] = []
+    num_seq = logits.shape[0]
+    for index in range(num_seq):
+        if (
+            sampling_params[index].logprobs is None or
+            not sampling_params[index].logprobs
```

Comment:
Because the following part is doing logprob sampling, IIUC this condition should be `sampling_params[index].logprobs` being `True` right?

Reply:
Hello @zxybazh! Thank you for review! Yes, it is a bug, I've fixed condition in correct way